mantle/cmd/ore/gcloud/upload.go

```diff
@@ -56,6 +57,7 @@ func init() {
 	cmdUpload.Flags().StringVar(&uploadWriteUrl, "write-url", "", "output the uploaded URL to the named file")
 	cmdUpload.Flags().StringVar(&uploadImageFamily, "family", "", "GCP image family to attach image to")
 	cmdUpload.Flags().StringVar(&uploadImageDescription, "description", "", "The description that should be attached to the image")
+	cmdUpload.Flags().BoolVar(&uploadJustUpload, "just-upload", false, "Don't create an image, just upload to GS")
```

Comment:
Bikeshed: `--upload-file-only`? And in cosa, we could make it imply `--upload` so that one doesn't have to specify both.

Reply:
Alternatively it could be `--create-image` defaulting to `true`

Reply:
@arithx i think I like that better. Will go with that unless anyone objects.

Reply:
ok I've pivoted this to `--create-image`.